### DISCUSSION TITLE
Don't go back to the main menu when invalid input

### DISCUSSION THREAD

**Discussion by dav1312** [ThioJoe/YT-Spammer-Purge#488] [Ideas]
It's very annoying having to do all the process again (and waiting for all of the comments to be scanned again) just because you misclicked or misspelled something, just ask again until a valid input is given.

![image](https://user-images.githubusercontent.com/63931154/152163509-327a3369-92e2-4525-a51b-70dbd8c80aed.png)

And instead of "to do nothing simply press enter" just change it to X to go back to the main menu just like in other prompt
![image](https://user-images.githubusercontent.com/63931154/152163031-4038c812-3eaf-42bf-b8e5-b2140b45267c.png)

**Comment by Firecul** [comment]
Agreed, I'd like it to give you the option to retype or correct yourself instead of just quitting out also.

**Comment by Firecul** [comment]
Seems to be fixed now 👍 
![WindowsTerminal_Windows_PowerShell (25)](https://user-images.githubusercontent.com/34950371/153735257-54282d1a-b737-4349-a0ce-ad5cffeea9d3.png)

**Comment by ThioJoe** [comment]
Yea this has been implemented into 2.16 Beta
